appinventor/components/src/com/google/appinventor/components/runtime/CloudDB.java

```diff
@@ -264,7 +285,19 @@ public void run() {
         try {
           String returnValue = jedis.get(accountName+projectID+tag);
           if (returnValue != null) {
-            value.set(returnValue);
+            try {
+              JSONArray valueJsonList = new JSONArray(returnValue);
```

Comment:
Normal lists should also be valid values. Currently, a list is represented as a list within a list with an additional entry in the super list.